common/lib/xmodule/xmodule/js/src/video/09_video_caption.js

```diff
@@ -110,12 +111,18 @@
                     '</div>'
                 ].join('');
 
-                var template = [
-                    '<div class="subtitles" role="region" id="transcript-' + this.state.id + '">',
-                        '<h3 id="transcript-label-' + this.state.id + '" class="transcript-title sr"></h3>',
-                        '<ol id="transcript-captions" class="subtitles-menu" lang="' + this.state.lang + '"></ol>',
-                    '</div>'
-                ].join('');
+                var template = HtmlUtils.interpolateHtml(
+                        HtmlUtils.HTML([
+                            '<div class="subtitles" role="region" id="transcript-{courseId}">',
+                                '<h3 id="transcript-label-{courseId}" class="transcript-title sr"></h3>',
+                                '<ol id="transcript-captions" class="subtitles-menu" lang="{courseLang}"></ol>',
+                            '</div>'
+                        ].join('')),
+                        {
+                            courseId: this.state.id,
+                            courseLang: this.state.lang
+                        }
+                    ).toString();
 
                 this.loaded = false;
                 this.subtitlesEl = $(template);
```

Comment:
The latter is out of scope, but I'll do it because it'll never get done otherwise.